doc/nrf/releases/release-notes-1.6.0.rst

```diff
@@ -0,0 +1,643 @@
+
+* nrf_cc3xx_platform/nrf_cc3xx_mbedcrypto:
+
+  * Added low-level APIs for managing and using hardware unique keys located in the KMU peripheral, or flash + K_DR, when no KMU is available.
+  * Added platform APIs for ``hmac_drbg``.
+  * Updated the used Mbed TLS version to 2.26.0 to align with upstream TF-M.
+  * For full information, see :ref:`crypto_changelog_nrf_cc3xx_platform` and :ref:`crypto_changelog_nrf_cc3xx_mbedcrypto`.
+
+nRF IEEE 802.15.4 radio driver
+++++++++++++++++++++++++++++++
+
+* Added production support for nRF5340 in multiprotocol configuration (IEEE 802.15.4 and Bluetooth Peripheral Role).
+
+MCUboot
+=======
+
+The MCUboot fork in |NCS| (``sdk-mcuboot``) contains all commits from the upstream MCUboot repository up to and including ``2fce9769b1``, plus some |NCS| specific additions.
+
+The code for integrating MCUboot into |NCS| is located in :file:`ncs/nrf/modules/mcuboot`.
+
+The following list summarizes the most important changes inherited from upstream MCUboot:
+
+* Added support for indicating serial recovery through LED.
+* Made the debounce delay of the serial detect pin state configurable.
+* Added support for Mbed TLS ECDSA for signatures.
+* Added an option to use GPIO PIN to enter to USB DFU class recovery.
+* Added an optional check that prevents attempting to boot an image built for a different ROM address than the slot it currently resides in.
+  The check is enabled if the image was signed with the ``IMAGE_F_ROM_FIXED`` flag.
+
+nrfx
+====
+
+See the `Changelog for nrfx 2.5.0`_ for detailed information.
+
+Mcumgr
+======
+
+The mcumgr library contains all commits from the upstream mcumgr repository up to and including snapshot ``5c5055f5a``.
+
+The following list summarizes the most important changes inherited from upstream mcumgr:
+
+* Fixed an issue with the file system management failing to open files due to missing initializations of ``fs_file_t`` structures.
+* Fixed an issue where multiple SMP commands sent one after the other would corrupt CBOR payload.
+* Fixed a problem where mcumgr over shell would stall and wait for retransmissions of frames.
+
+Zephyr
+======
+
+The Zephyr fork in |NCS| (``sdk-zephyr``) contains all commits from the upstream Zephyr repository up to and including ``730acbd6ed`` (``v2.6.0-rc1``), plus some |NCS| specific additions.
+
+For a complete list of upstream Zephyr commits incorporated into |NCS| since the most recent release, run the following command from the :file:`ncs/zephyr` repository (after running ``west update``):
+
+.. code-block:: none
+
+   git log --oneline v2.6.0-rc1 ^v2.4.99-ncs1
+
+For a complete list of |NCS| specific commits, run:
+
+.. code-block:: none
+
+   git log --oneline manifest-rev ^v2.6.0-rc1
+
+The current |NCS| release is based on Zephyr v2.6.0-rc1.
+See the :ref:`zephyr:zephyr_2.6` Release Notes for an overview of the most important changes inherited from upstream Zephyr.
+
```

Comment:
Feel free to fix language and references - I suggest that this should be noted here and reference made to the later section.

```suggestion
Important Note: This version of |NCS| has incorporated the use of new Zephyr Workqueue APIs and avoids the use of the deprecated APIs.  For specific information on what has changed and guidance on taking the new APIs into use, see the section "Zephyr Workqueue API Migration" at the bottom of these release notes.
```

Reply:
no need to change this as the migration guide starts right after this line in the original release notes PR.